.github/workflows/modsec-config.yml

```diff
@@ -0,0 +1,68 @@
+name: "Get Modsec config."
+
+on:
+  workflow_call:
+    outputs:
+      development:
+        description: "Modsec Config. for Development"
+        value: ${{ jobs.get_modsec_config.outputs.development }}
+      demo:
+        description: "Modsec Config. for Development"
+        value: ${{ jobs.get_modsec_config.outputs.demo }}
+      staging:
+        description: "Modsec Config. for Development"
+        value: ${{ jobs.get_modsec_config.outputs.staging }}
+      production:
+        description: "Modsec Config. for Production"
+        value: ${{ jobs.get_modsec_config.outputs.production }}
+
+jobs:
+  get_modsec_config:
```

Comment:
Hey buddy, `get_` is thrown around a lot, a job could be called `modsec_config` without a task, it would read better in output values.